pages/tidal tools/machine_stats.md

```diff
@@ -32,14 +32,13 @@ Migrations." %}
 
 ### Introduction
 
-This is a simple and effective way to gather machine stats (RAM, Storage, CPU)
-from a Windows Server environment. For better results, we recommend using it
-with
-[WinRM](https://docs.microsoft.com/en-us/windows/win32/winrm/installation-and-configuration-for-windows-remote-management)
-to `Invoke-Command` across your servers. However, it is also possible to
-execute in non-WinRM environments, though not all of the stats will be
-available to gather in such case. As a result it outputs JSON data which can be
-securely sent to your Tidal Migrations workspace using the `tidal` command.
+Machine Stats for Windows is a simple and effective way to gather machine stats (RAM, Storage, CPU) from a Windows Server environment. 
+For better results, we recommend using it with [WinRM](https://docs.microsoft.com/en-us/windows/win32/winrm/installation-and-configuration-for-windows-remote-management)
+to `Invoke-Command` across your servers.
+
+If WinRM is not the ideal solution for you, we offer an alternative approach backed by WMI. See [this section](#gather-machine-stats-without-winrm) for more information.
+
+Machine Stats for Windows outputs JSON data which can be securely sent to your Tidal Migrations workspace using the `tidal` command.
```

Comment:
Maybe we can specify the command? or alternative just link to Tidal Tools. 

"..... sent to your Tidal Migrations workspace using the Tidal Tools"
"..... sent to your Tidal Migrations workspace using the `tidal sync` command.

What do you think?

Reply:
Hey Mario, I'm not 100% sure - I don't really want to get into explaining commands in the intro as I think it's a bit early. I agree that this sentence is out of place though. I'm thinking something more high level like:

Machine Stats for Windows outputs JSON data, which can be saved and uploaded to the Tidal Migrations Platform, or piped there directly using Tidal Tools.  

Then directly after this will be the updated 'how to run MS4W' section that explains in detail how to do both of those things. WDYT?

Reply:
I realize this is basically your first option there, sorry bit late in the day for me.